app/assets/stylesheets/global/_base.scss

```diff
@@ -157,48 +157,6 @@ video {
   max-width: 100%;
 }
 
-.video-wrapper {
```

Comment:
Is there anything more satisfying as deleting close to 50 lines of code in one fell swoop? Nay, there is not.